### PR TITLE
Bug fix/ui

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by m4rtint** [comment]
Things to test:
- Changed Main menu button hit box to be expanded
- Level selector, click image to select the map
- Refactored LevelSelector.cs, SceneInformation.cs

**Comment by m4rtint** [comment]
[Menu button hit boxes in wrong places](https://trello.com/c/712fKC85/139-menu-button-hit-boxes-in-wrong-places)

**Comment by m4rtint** [comment]
[Npc Gazebo level wont load](https://trello.com/c/Xq59dqsE/128-npc-gazebo-level-wont-load)

**Comment by m4rtint** [comment]
[Part of the Office Level looks weird](https://trello.com/c/PWsgmm2G/129-part-of-the-office-level-looks-weird)

File: Assets/Script/UI/LevelSelection.cs [hunk 1]
```diff
@@ -3,64 +3,47 @@
...
 
     /// <summary>
     /// Find the Dropdown and set the levels
     /// </summary>
     private void LevelInitializer()
     {
-        //m_levelSelector.
-
-        m_listOfScenes = System.Enum.GetNames(typeof(SCENENAME)).ToList();
+        var m_listOfScenes = System.Enum.GetNames(typeof(SCENENAME)).ToList();
```
Comment (Gursaanj) [inline Assets/Script/UI/LevelSelection.cs]: Should we declaring this as a Local List variable instead on just a Var (just to be consistent with all our other code)

Comment (m4rtint) [reply Assets/Script/UI/LevelSelection.cs]: Not my code

Comment (Gursaanj) [reply Assets/Script/UI/LevelSelection.cs]: haha fair enough, will fix it in the next refactor

**Pull request merged**